src/main/java/com/virtudoc/web/controller/CurrentAppointmentsController.java

```diff
@@ -18,42 +20,48 @@
     @GetMapping("/notifications")
     String getNotifications(Model model) {
         //Manually add 3 appointments
-        Appointment apt1 = new Appointment();
-        apt1.setEmail("test@test.com");
-        apt1.setPatientName("John Smith");
-        apt1.setSymptoms("Covid");
-        apt1.setDoctorName("Dr. Mary Jane");
-        apt1.setLocation("Kennesaw Side Building");
-        apt1.setDate("2PM Feb 3");
-        apt1.setReasonForVisit("Covid-19 Test");
-
-        Appointment apt2 = new Appointment();
-        apt2.setEmail("test@test.com");
-        apt2.setPatientName("Jane Smith");
-        apt2.setSymptoms("Covid");
-        apt2.setDoctorName("Dr. Phil Johnson");
-        apt2.setLocation("Kennesaw Main Building");
-        apt2.setDate("3PM Feb 4");
-        apt2.setReasonForVisit("Covid-19 Test");
-
-        Appointment apt3 = new Appointment();
-        apt3.setEmail("test@test.com");
-        apt3.setPatientName("John Smith");
-        apt3.setSymptoms("Covid");
-        apt3.setDoctorName("Dr. Mary Jane");
-        apt3.setLocation("Marietta Main Building");
-        apt3.setDate("1PM Feb 7");
-        apt3.setReasonForVisit("Annual Exam");
-
-        model.addAttribute("appointments", Arrays.asList(
-                apt1, apt2, apt3
-        ));
-
+//        Appointment apt1 = new Appointment();
+//        apt1.setEmail("test@test.com");
+//        apt1.setPatientName("John Smith");
+//        apt1.setSymptoms("Covid");
+//        apt1.setDoctorName("Dr. Mary Jane");
+//        apt1.setLocation("Kennesaw Side Building");
+//        apt1.setDate("2PM Feb 3");
+//        apt1.setReasonForVisit("Covid-19 Test");
+//
+//        Appointment apt2 = new Appointment();
+//        apt2.setEmail("test@test.com");
+//        apt2.setPatientName("Jane Smith");
+//        apt2.setSymptoms("Covid");
+//        apt2.setDoctorName("Dr. Phil Johnson");
+//        apt2.setLocation("Kennesaw Main Building");
+//        apt2.setDate("3PM Feb 4");
+//        apt2.setReasonForVisit("Covid-19 Test");
+//
+//        Appointment apt3 = new Appointment();
+//        apt3.setEmail("test@test.com");
+//        apt3.setPatientName("John Smith");
+//        apt3.setSymptoms("Covid");
+//        apt3.setDoctorName("Dr. Mary Jane");
+//        apt3.setLocation("Marietta Main Building");
+//        apt3.setDate("1PM Feb 7");
+//        apt3.setReasonForVisit("Annual Exam");
+//
+//        model.addAttribute("appointments", Arrays.asList(
+//                apt1, apt2, apt3
+//        ));
 
         //Pull appointments from DB
-//        List<Appointment> allAppointments = service.listAll();
-//        model.addAttribute("appointments", allAppointments);
+        List<Appointment> allAppointments = service.listAll();
+        model.addAttribute("appointments", allAppointments);
 
         return "current_appointments";
     }
+
+    //Delete by id
+    @GetMapping("/notifications/delete/{id}")
```

Comment:
Nit: Typically non-read operations in a CRUD controller use POST instead of GET to protect against CSRF and XSS attacks. Switching to a POST method would require re-implementing the delete button as part of a form (or adding some JS code), so up to you if you want to implement that or not.